packages/server/src/admin/client.test.ts

```diff
@@ -58,4 +63,42 @@ describe('Client admin', () => {
       .send({ foo: 'bar' });
     expect(res4.status).toBe(400);
   });
+
+  test('Create client as superadmin - verify projectID', async () => {
+    const { project } = await createTestProject();
+
+    // As a superadmin, create a new client
+    const superAdminAccessToken = await initTestAuth({ superAdmin: true });
+    const res = await request(app)
+      .post('/admin/projects/' + project.id + '/client')
+      .set('Authorization', 'Bearer ' + superAdminAccessToken)
+      .send({
+        name: 'Another one',
```

Comment:
We should keep these names descriptive. Feel free to reach out if you want to talk through some potential names